x-pack/plugins/triggers_actions_ui/public/application/lib/rule_api/load_execution_log_aggregations.ts

```diff
@@ -119,6 +123,7 @@ export const loadGlobalExecutionLogAggregations = async ({
         // whereas data grid sorts are 0 indexed.
         page: page + 1,
         sort: sortField.length ? JSON.stringify(sortField) : undefined,
+        namespaces: namespaces ? JSON.stringify(namespaces) : namespaces,
```

Comment:
nitpick: would it be clearer to say `namespaces: namespaces ? JSON.stringify(namespaces) : undefined,`, like how we're doing it the line above